Quadriga/src/main/java/edu/asu/spring/quadriga/rest/SearchNetworkRestController.java

```diff
@@ -108,10 +113,19 @@
             return new ResponseEntity<String>(errorMsg, HttpStatus.NOT_FOUND);
         }
 
+        // Fetch ConceptPower entries related to the conceptId
+        ConceptpowerReply reply = conceptpowerConnector.getById(conceptId);
+        List<String> alternativeIdsForConcept = null;
+        if (reply != null && reply.getConceptEntry().size() > 0) {
+            alternativeIdsForConcept = reply.getConceptEntry().get(0).getAlternativeIdList();
+        }
+        if(alternativeIdsForConcept == null){
+            alternativeIdsForConcept = new ArrayList<String>();
+        }  
```

Comment:
Both the controllers SearchNetworkRestController.java and NetworkSearchController.java use the  service NetworkTransformationManager. Agree there is code duplication wrt finding alternative ids for a concept id. I thought fetching and providing information to the service layer is the work of the controller and the service layer is responsible for transformations and dealing with database layer. Wouldn't shifting the functionality (fetching alternative ids) to service layer lead to additional overhead in the service layer?